map/src/test/resources/openlr/map/mockdb/SampleSimpleMockDatabase.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<SimpleMockedMapDatabase>
+    <Node>
+        <id>1</id>
+        <longitude>13.476134</longitude>
+        <latitude>52.50507</latitude>
+    </Node>
+
+    <Node>
+        <id>2</id>
+        <longitude>13.47671</longitude>
+        <latitude>52.504942</latitude>
+    </Node>
+
+    <Node>
+        <id>3</id>
+        <longitude>13.477137</longitude>
+        <latitude>52.505390</latitude>
+    </Node>
+
+    <Line>
+        <id>1</id>
+        <start>1</start>
+        <end>2</end>
+        <frc>0</frc>
+        <fow>1</fow>
+        <ShapePoint>
```

Comment:
I'm a bit confused.  I was expecting that the lines would have at least 2 shape points.

Reply:
this is an intermediate point the other two points are coming from start and end node

Reply:
9921bc46eaef1f2b022379c9e24cba52db5b1735